packages/contracts/test/multi_sig_with_time_lock.ts

```diff
@@ -153,39 +119,29 @@ describe('MultiSigWalletWithTimeLock', () => {
             after(async () => {
                 await blockchainLifecycle.revertAsync();
             });
-            before('deploy a wallet', async () => {
+            let txId: BigNumber;
+            const newSecondsTimeLocked = new BigNumber(0);
+            before('deploy a wallet, submit transaction to change timelock, and confirm the transaction', async () => {
                 multiSig = await MultiSigWalletWithTimeLockContract.deployFrom0xArtifactAsync(
                     artifacts.MultiSigWalletWithTimeLock,
                     provider,
                     txDefaults,
                     owners,
-                    SIGNATURES_REQUIRED,
+                    REQUIRED_APPROVALS,
                     SECONDS_TIME_LOCKED,
                 );
-                multiSigWrapper = new MultiSigWrapper((multiSig as any) as MultiSigWalletContract);
+                multiSigWrapper = new MultiSigWrapper(multiSig, zeroEx);
```

Comment:
Pass in `provider` here instead of `zeroEx` and instantiate a `Web3Wrapper` inside it.